vehicle/embed.go

```diff
@@ -5,13 +5,15 @@ import (
 )
 
 type embed struct {
-	Title_       string           `mapstructure:"title"`
-	Icon_        string           `mapstructure:"icon"`
-	Capacity_    float64          `mapstructure:"capacity"`
-	Phases_      int              `mapstructure:"phases"`
-	Identifiers_ []string         `mapstructure:"identifiers"`
-	Features_    []api.Feature    `mapstructure:"features"`
-	OnIdentify   api.ActionConfig `mapstructure:"onIdentify"`
+	Title_            string           `mapstructure:"title"`
+	Icon_             string           `mapstructure:"icon"`
+	Capacity_         float64          `mapstructure:"capacity"`
+	Phases_           int              `mapstructure:"phases"`
+	DefaultTargetSoc_ int              `mapstructure:"targetSoc"`
+	MinSoc_           int              `mapstructure:"minSoc"`
+	Identifiers_      []string         `mapstructure:"identifiers"`
+	Features_         []api.Feature    `mapstructure:"features"`
+	OnIdentify        api.ActionConfig `mapstructure:"onIdentify"`
```

Comment:
Brauchts das jetzt noch? Für den Modus? Den könnten wir dann auch direkt noch in ein API setzen.